src/main/java/nl/ycn/coaching/controller/HrController.java

```diff
@@ -0,0 +1,85 @@
+package nl.ycn.coaching.controller;
+
+
+import net.bytebuddy.utility.RandomString;
+import nl.ycn.coaching.database.AppUserService;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Controller
+public class HrController {
+
+	private String randomPassword="";
```

Comment:
Dit is niet goed: randomPassword variabele wordt gedeeld met alle gebruikers. Kan nooit een attribuut van de klasse zijn.